zhang_mcfarlane/zm_conv_convtran.F90

```diff
@@ -68,9 +68,6 @@ subroutine zm_conv_convtran_run(ncol, pver, &
 
    real(kind_phys), intent(in) :: dpdry(:,:)    ! Delta pressure between interfaces        (ncol,pver)
 
-   real(kind_phys), intent(in) :: dt                      ! 2 delta t (model time increment)
```

Comment:
I presume dt is just not used in convtran?

Reply:
That is correct - when I went to see if the dt needed adjusting in this routine, I discovered it was not used, so I removed it.